source/common/buffer/buffer_impl.cc

```diff
@@ -25,7 +25,7 @@ void OwnedImpl::addImpl(const void* data, uint64_t size) {
   bool new_slice_needed = slices_.empty();
   while (size != 0) {
     if (new_slice_needed) {
-      slices_.emplace_back(size);
+      slices_.emplace_back(Slice(size, account()));
```

Comment:
Prefer account_ instead of account() when calling Slice constructors for clarity.